tyrant/cogs/lemon_say.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import random
+
+from discord.ext import commands
+from discord.ext.commands import Bot, Cog, Context
+
+from tyrant import constants
+
+log = logging.getLogger(__name__)
+
+
+class LemonSay(Cog):
+    """Post answer to generic yes/no questions"""
+
+    def __init__(self, bot:Bot):
+        """Initialize this cog with the Bot instance"""
+        self.bot = bot
+
+        self.RESPONSES = [
+            "Yep.",
+            "Absolutely!",
+            "Affirmative!",
+            "Sure.",
+            "Sure thing!",
+            "Okay.",
+            "Alright.",
+            "ROGER THAT",
+            "Of course!",
+            "Aye aye, cap'n!",
+            "Noooooo!!",
+            "Nope.",
+            "I don't think so.",
+            "Out of the question.",
+            "Huh? No.",
+            "Nah.",
+            "Naw.",
+            "Not likely.",
+            "No way, José.",
+            "Not in a million years.",
+            "Fat chance.",
+            "Certainly not.",
+            "NEGATORY.",
+            "Nuh-uh.",
+            ]
+
+    
+    @commands.command(aliases=('lemonsay','lsay',))
+    async def say(self, ctx:Context):
+        """send random answers to generic yes/no kind of question"""
```

Comment:
```suggestion
        """Send random answers to generic yes or no questions."""
```
First letter must be capitalized, punctuation in the end.